test/e2e_new/trigger_finalizer_test.go

```diff
@@ -101,7 +99,7 @@ func hasNoKafkaBrokerFinalizer() func(ctx context.Context, t feature.T) {
 		time.Sleep(time.Second * 20) // "eventually"
 		tr := triggerfeatures.GetTrigger(ctx, t)
 		for _, f := range tr.Finalizers {
-			require.NotEqual(t, f, triggerreconciler.FinalizerName, "%+v", tr)
+			require.NotEqual(t, f, "kafka.triggers.eventing.knative.dev", "%+v", tr)
```

Comment:
what's the need for this change?

Reply:
Full context is here: https://github.com/knative-extensions/eventing-kafka-broker/pull/3719#issuecomment-2034067621.
TL;DR; is, that this lead to an import an then registered an informer, which lead to a test failure in this case.